cips/cip-26.md

```diff
@@ -0,0 +1,64 @@
+    1. Celestia consensus nodes SHOULD update their software to accommodate this change prior to the agreed-upon block height.
+    1. Client applications interacting with the Celestia network SHOULD be updated to account for the faster block time, particularly in areas related to transaction confirmation and block finality.
+
+1. The default `ttl-num-blocks` parameter in the mempool configuration SHALL be increased from 5 to 12. This change is necessary to maintain consistency with the new block time and ensure that transactions remain in the mempool for a similar duration as before.
+    1. Current default: `ttl-num-blocks = 5`
+    1. New default: `ttl-num-blocks = 12`
+    1. This change SHALL NOT be implemented alongside the block time reduction. The default increase from 5 to 12 will occur when users upgrade to v3.0.0 and regenerate their config files. The block time reduction will happen one week later when the v2 to v3 activation height occurs. This approach ensures consistent behavior of the mempool across the network upgrade.
+    1. All validator nodes MUST update their configuration files to reflect this new `ttl-num-blocks` value before the agreed-upon implementation block height.
+
+1. Documentation and APIs related to block time and block production MUST be updated to reflect these changes.
+
+## Rationale
+
+The rationale for this change is to increase the throughput of the Celestia network by doubling the number of blocks produced per unit of time. This will reduce the time it takes for transactions to be finalized and improve the overall user experience on the network.
+
+The increase in `ttl-num-blocks` from 5 to 12 is necessary to maintain consistent mempool behavior with the new block time. This change ensures that transactions remain in the mempool for approximately 72 seconds (12 blocks times 6 seconds), which closely matches the previous behavior of about 60 seconds (5 blocks times 12 seconds).
+
+## Backwards Compatibility
+
+This is a breaking network upgrade and will require all participants to update their software to accommodate the new block time and `ttl-num-blocks`. The change in block time will not be backward compatible with the existing network, and all participants MUST be compliant by the agreed-upon implementation block height.
+
+## Test Cases
+
+This will be tested on Arabica devnet and Mocha testnet before going live on Celestia Mainnet Beta.
+
+## Security Considerations
+
+While the reduction in block time itself does not introduce significant new security risks to the network, there are important considerations:
+
+1. Participants should ensure that their systems are capable of handling the increased throughput from faster block times.
+1. The increase of `ttl-num-blocks` from 5 to 12 is crucial for maintaining the security and efficiency of the mempool:
```

Comment:
[question] Shouldn’t we increase the mempool size? My intuition is that we’re likely processing twice as many transactions in the same amount of time as before, so it might be worth considering a larger mempool to accommodate the increased volume.

Reply:
I'm resolving this based on [slack discussion](https://celestia-team.slack.com/archives/C02PFFCGMNW/p1729115035368559), but please feel free to reopen